docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,205 @@
+---
+id: integrating-with-other-libraries
+title: Integrating with other libraries
+permalink: docs/integrating-with-other-libraries.html
+---
+
```

Comment:
Could you add a one-paragraph summary for this page? Something that says that React can be used in any application, and summarizing two methods below.